tests/unit_tests/test_profile_validation.py

```diff
@@ -88,6 +88,12 @@ def test_profile_validation_valid_offset_none(self, mock_georaster_utils):
         )
         self.assert_response(response)
 
+    @patch('app.routes.georaster_utils')
+    def test_profile_validation_wrong_content_type(self, mock_georaster_utils):
+        prepare_mock(mock_georaster_utils)
+        response = self.test_instance.post('/rest/services/profile.json', headers=self.headers)
+        self.assert_response(response, expected_status=415)
+
```

Comment:
seems like the CI let this go (as before) to a 400, shouldn't we force the `Content-Type` to something wrong here so that we are sure to hit the right portion of the `if...else`?

Reply:
Strange was working locally, but your right it would be wiser here to be more explicit, I'll change the test.